### PR TITLE
fixes responsiveness

### LINKED ISSUE #4110: improve responsiveness of landing page
### Is your feature request related to a problem? Please describe

The landing page of CircuitVerse in the mobile version is not responsive design.

Check out the attached image. The buttons are not responsive.

### Describe the solution you'd like

Here is a preferred design for this page. However, the design is open for feedback as well.

|  Old design | New Design |
|------------|-------------|
|  |  |

### Are you working on this?

No

### PR DESCRIPTION
**Pull request by Malavi1**
Fixes #4110

#### Describe the changes you have made in this PR -
I just fixed the media queries of the 3 buttons.

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -49,30 +49,34 @@
...
   .home-image-cover {
     display: none;
   }
 }
 
-@media(max-width: 400px) {
+@media (max-width: 400px) {
   .home-tertiary-button-container {
     text-align: center;
   }
```
Comment (niladrix719) [inline app/assets/stylesheets/home.scss:68]: add space here after line 68

**Review by niladrix719 — CHANGES_REQUESTED** [review]

**Comment by Malavi1** [comment]
@tachyons could you please review my PR

**Comment by tanmoysrt** [comment]
@Malavi1 this seems to be ready for merging. Fix the code climate issue.

**Comment by Malavi1** [comment]
@tanmoysrt  could you please help me with this

**Comment by tanmoysrt** [comment]
> @tanmoysrt  could you please help me with this

Fix the codeclimate issue

**Comment by tanmoysrt** [comment]
Hi @Malavi1 can you resolve the code climate issues?

**Comment by Malavi1** [comment]
Yes I did but after also I am getting this error. 
Could you please help me with this?

**Comment by Malavi1** [comment]
> hi @Malavi1 line no. 69 is containing a space can you please remove it, i think that will fix the code climate issue

As for the flow it needs a white space right?

**Comment by niladrix719** [comment]
@Malavi1 the code climate issue still persists, you will need to add the vertical space between the function declarations and also make sure there is no space horizontally on line no. 69

**Review by niladrix719 — APPROVED** [review]

**Comment by Malavi1** [comment]
@tanmoysrt review it once

**Review by tanmoysrt — APPROVED** [review]
Looks good !
cc @vedant-jain03

**Pull request merged**